pkg/apis/application/v1alpha1/types.go

```diff
@@ -468,6 +474,12 @@ type Info struct {
 type SyncPolicy struct {
 	// Automated will keep an application synced to the target revision
 	Automated *SyncPolicyAutomated `json:"automated,omitempty" protobuf:"bytes,1,opt,name=automated"`
+	// Validate does `kubectl apply --validate=false`. Defaults to true.
+	Validate *bool `json:"validate,omitempty" protobuf:"bytes,2,opt,name=validate"`
```

Comment:
Instead of a bool here, can we make this a list of options such that it is consistent with the annotations?

Reply:
How about ?
```
Options *SyncOptions `json...`
```
```
type SyncOptions struct {
  Prune *bool
}
```